ignite/cmd/node_tx_bank_send.go

```diff
@@ -0,0 +1,109 @@
+package ignitecmd
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ignite-hq/cli/ignite/pkg/cliui"
+	"github.com/ignite-hq/cli/ignite/pkg/cosmosclient"
+	"github.com/spf13/cobra"
+)
+
+func NewNodeTxBankSend() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "send [from_account_or_address] [to_account_or_address] [amount]",
+		Short: "Send funds from one account to another.",
+		RunE:  nodeTxBankSendHandler,
+		Args:  cobra.ExactArgs(3),
+	}
+
+	c.Flags().AddFlagSet(flagSetHome())
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+	c.Flags().AddFlagSet(flagSetAccountPrefixes())
+	c.Flags().AddFlagSet(flagSetKeyringDir())
+	c.Flags().AddFlagSet(flagSetTxFrom())
+	c.Flags().AddFlagSet(flagSetGenerateOnly())
+	c.Flags().AddFlagSet(flagSetGasFlags())
+
+	return c
+}
+
+func nodeTxBankSendHandler(cmd *cobra.Command, args []string) error {
+	var (
+		fromAccountInput = args[0]
+		toAccountInput   = args[1]
+		amount           = args[2]
+		home             = getHome(cmd)
+		prefix           = getAddressPrefix(cmd)
+		from             = getFrom(cmd)
+		node             = getRPC(cmd)
+		keyringBackend   = getKeyringBackend(cmd)
+		keyringDir       = getKeyringDir(cmd)
+		generateOnly     = getGenerateOnly(cmd)
+		gas              = getGas(cmd)
+		gasPrices        = getGasPrices(cmd)
+	)
+
+	session := cliui.New()
+	defer session.Cleanup()
+
+	session.StartSpinner("Sending transaction...")
+
+	client, err := cosmosclient.New(cmd.Context(),
+		cosmosclient.WithAddressPrefix(prefix),
+		cosmosclient.WithHome(home),
+		cosmosclient.WithKeyringBackend(keyringBackend),
+		cosmosclient.WithKeyringDir(keyringDir),
+		cosmosclient.WithNodeAddress(node),
+		cosmosclient.WithGas(gas),
+		cosmosclient.WithGasPrices(gasPrices),
+	)
+	if err != nil {
+		return err
+	}
+
+	// If from flag is missing, check if the "from account" argument is an account name and use that instead
+	if from == "" {
+		fromInputIsAccount, err := client.AccountExists(fromAccountInput)
+		if err != nil {
+			return err
+		}
+		if fromInputIsAccount {
+			from = fromAccountInput
+		} else {
+			return fmt.Errorf("\"--%s\" flag is required when from address is not an account name", flagFrom)
+		}
+	}
+
+	fromAddress, err := client.Bech32Address(fromAccountInput)
+	if err != nil {
+		return err
+	}
+
+	toAddress, err := client.Bech32Address(toAccountInput)
+	if err != nil {
+		return err
+	}
+
+	coins, err := sdk.ParseCoinsNormalized(amount)
+	if err != nil {
+		return err
+	}
+
+	if generateOnly {
+		tx, err := client.BankSendGenerateOnly(fromAddress, toAddress, coins, from)
+		if err != nil {
+			return err
+		}
+
+		return session.Println(tx)
+	}
+
+	if err := client.BankSend(fromAddress, toAddress, coins, from); err != nil {
+		return err
+	}
+
+	session.Println("Transaction broadcast successful!")
```

Comment:
```suggestion
        session.StopSpinner("Sending transaction...")
	session.Println("Transaction broadcast successful!")
```
At this point spinner should be stopped manually